src/module.js

```diff
@@ -266,7 +266,9 @@ const PARTICIPANT_PROPS = {
       if (callObject._preloadCache.subscribeToTracksAutomatically) {
         return false;
       }
-      if ([true, false, 'avatar'].includes(subs)) {
+      let validPrimitiveValues = [true, false];
+      !isReactNative() && validPrimitiveValues.push('avatar');
+      if (validPrimitiveValues.includes(subs)) {
```

Comment:
i love reading your code because i never would have thought to do the check this way.

Reply:
i'll take that as a compliment, though i'm not sure it is haha

Reply:
Ah! It is! I'm sorry it didn't come off that way.